samples/mapzen-places-api-sample/src/main/res/layout/activity_autocomplete.xml

```diff
@@ -1,18 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:orientation="vertical"
     android:paddingBottom="@dimen/activity_vertical_margin"
     android:paddingLeft="@dimen/activity_horizontal_margin"
     android:paddingRight="@dimen/activity_horizontal_margin"
     android:paddingTop="@dimen/activity_vertical_margin"
     >
 
-  <fragment
-      android:id="@+id/place_autocomplete_fragment"
-      android:name="com.mapzen.places.api.ui.PlaceAutocompleteFragment"
```

Comment:
You read my mind...I was also trying to show where to handle callbacks. Updated `PlaceAutocompleteFragment`, restored the fragment to the demo, and added stubs for the callbacks (though with this example they are slightly different than before aka an explicit listener is set instead of callbacks received in `onActivityResult`)